### PR TITLE
adding simple fio profile for IO run and yaml file to accept input

### PR DESCRIPTION
**Pull request by Naresh-ibm**
Adding a simple fio profile which can run a quick IO without taking
much time as the default profile "fio-mixed.job" file takes.
And added a yaml file to accept the inputs like disk on which it
can run the IO and a job profile file.
altered the fiotest.py to run with yaml file.

Signed-off-by: Naresh Bannoth <nbannoth@in.ibm.com>

### CODE REVIEW

**Comment by Naresh-ibm** [comment]
[fio-simple-profile.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/802806/fio-simple-profile.txt)

**Comment by Naresh-ibm** [comment]
Changes are made that asked in 
https://github.com/avocado-framework-tests/avocado-misc-tests/pull/283

File: perf/fiotest.py.data/fio-simple.job [hunk 1]
```diff
@@ -0,0 +1,73 @@
+; fio-mixed.job
```
Comment (PraveenPenguin) [inline perf/fiotest.py.data/fio-simple.job]: i think it still need to addressed from previous PR

Comment (PraveenPenguin) [inline perf/fiotest.py.data/fio-simple.job]: changed to fio-simple.job

**Comment by PraveenPenguin** [comment]
@narasimhan-v thanks

**Pull request merged**